Tophat.xcodeproj/project.pbxproj

```diff
@@ -1208,6 +1211,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_USER_SCRIPT_SANDBOXING = YES;
```

Comment:
We'll need to discard this change (there's a second instance of it below as well) for now as this would prevent SwiftLint from working in the build phases.  Needs further exploration.